gradle-xmake-plugin/src/main/resources/lua/install_artifacts.lua

```diff
@@ -38,6 +47,15 @@ function _install_artifacts(libsdir, installdir, targets, arch)
     end
     for _, target in ipairs(targets) do
         os.vcp(path.join(libsdir, path.filename(target:targetfile())), installdir)
+        
+        -- install target packages
+        local package_installdirs = _get_from_target(target, "linkdirs", "package::*")
+        for _, package_installdir in ipairs(installdirs) do
+            local files = os.files(path.translate(format("%s/*.so", package_installdir)))
```

Comment:
it maybe will copy incorrect system and unused libs.

you can refer https://github.com/xmake-io/xmake/blob/f9196dfa7073e3c3511230072d606c89f28a075d/xmake/modules/target/action/install/main.lua#L125

Reply:
So wouldn't it be cleaner to add an opt argument to `actions.install.main` to define install dir ? And use the install action script in `install_artifacts.lua`